FWCore/Concurrency/scripts/edmStreamStallGrapher.py

```diff
@@ -5,6 +5,7 @@
 import sys
 from collections import defaultdict
 import six
+import yaml 
```

Comment:
Given that yaml is not part of the standard python package, we should avoid requiring all calls to the script requiring it to load.